src/net/sourceforge/kolmafia/textui/command/AbsorbCommand.java

```diff
@@ -52,15 +55,19 @@ public void run(final String cmd, String parameters) {
       return;
     }
 
-    // Absorb the item
-    RequestThread.postRequest(
-        new GenericRequest("inventory.php?absorb=" + itemId + "&ajax=1", false));
+    // Absorb the item(s)
+    for (int i = 0; i < count; i++) {
+      RequestThread.postRequest(
+          new GenericRequest("inventory.php?absorb=" + itemId + "&ajax=1", false));
+    }
```

Comment:
Is it necessary to allocate a new request each time through the loop, or could you make one and reuse it?
I don't feel strongly about it, but I'd like to hear your rationale if you do feel strongly.

Reply:
sure, I can re-use it. I didn't spot that it was creating a new object.